lib/matplotlib/rcsetup.py

```diff
@@ -988,7 +988,12 @@ def validate_webagg_address(s):
     raise ValueError("'webagg.address' is not a valid IP address")
 
 
-validate_axes_titlelocation = ValidateInStrings('axes.titlelocation', ['left', 'center', 'right'])
+_validate_axes_titlelocation = ValidateInStrings('axes.titlelocation',
```

Comment:
Ah yes. Sorry I missed that; it'll need to start public for now (and get properly deprecated later).